docs/REFERENCE-environment_variables.md

```diff
@@ -1,81 +1,82 @@
+| DB_USE_SSL                                       | Boolean value to determine whether database connections should use SSL. _Default_: false.                                                                                                                                                                                                                                                                                                   |
+| DEBUG_SCALING                                    | Emit console.log on events related to scaling issues. _Default_: false.                                                                                                                                                                                                                                                                                                                     |
+| DEFAULT_SERVICE                                  | Default SMS service. _Options_: twilio, nexmo, fakeservice.                                                                                                                                                                                                                                                                                                                                 |
+| DEFAULT_ORG                                      | Set only with FIX_ORGLESS. Set to integer organization.id corresponding to the organization you want orgless users to be assigned to.                                                                                                                                                                                                                                                       |
+| DEV_APP_PORT                                     | Port for development Webpack server. Required for development.                                                                                                                                                                                                                                                                                                                              |
+| DISABLE_DB_LOG                                   | Disable saving error log info to `log` table                                                                                                                                                                                                                                                                                                                                                |
```

Comment:
this is the one-line change in this file -- the rest is changed by `prettier`

Reply:
Why would we want this as an env var? Curious why anyone would want to turn this feature off

Reply:
This adds a lot of data (and writes) into the database that is arguably better examined in actual log data (vs. a db table).  I think at some point we might even want this as a default, but Jeff's use-case with MediaUrls should probably be addressed in some way or would be an exception (we could skip the condition based on DISABLE_DB_LOG when the MediaUrl is present)